cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricStore.java

```diff
@@ -130,6 +130,14 @@ public Cube get() {
       // i.e. for service only
       ImmutableList.of(Constants.Metrics.Tag.NAMESPACE, Constants.Metrics.Tag.APP,
                        Constants.Metrics.Tag.SERVICE)));
+    // worker
+    aggs.add(new DefaultAggregation(
```

Comment:
indentation is fine, the able comment `` for service only`` is for a parameter, that's why it seems off , but isn't.